app/containers/Header/index.js

```diff
@@ -149,21 +154,45 @@ const SearchWrap = styled(Box)`
   }
 `;
 
-const renderPages = ({ match, onClick, align }) =>
+const navButtonOnClick = ({ match, onClick, align, locale }) =>
   PAGES &&
   Object.values(PAGES)
     .filter(page => page.primary)
-    .map(page => (
-      <ButtonNavPrimary
-        key={page.key}
-        active={page.key === match}
-        disabled={page.key === match}
-        align={align}
-        onClick={() => onClick(page.key)}
-      >
-        <FormattedMessage {...rootMessages.page[page.key]} />
-      </ButtonNavPrimary>
-    ));
+    .map(page =>
+      page.key === 'download' ? (
+        <a
+          href={XPATHS.download[locale] || XPATHS.download[DEFAULT_LOCALE]}
+          target="_blank"
+          rel="noopener noreferrer"
+          key={page.key}
+        >
+          <ButtonNavPrimary>
+            <FormattedMessage {...rootMessages.page[page.key]} />
+            {/* prettier-ignore */}
+            <Share
```

Comment:
Noting that we are using the "Share" icon here but the "NewWindow" icon elsewhere (home page sections). Ideally this should be consistent.

Reply:
replacing now...